src/adapters/xdb_mnesia_adapter.erl

```diff
@@ -89,6 +89,7 @@ in_transaction(_Repo) ->
   mnesia:is_transaction().
 
 %% @hidden
```

Comment:
Let's remove this then

Reply:
Removing this `hidden` still fails dialyzer check :(